src/ossia/network/base/node_functions.hpp

```diff
@@ -165,5 +165,10 @@ void expand_ranges(std::string& str);
 OSSIA_EXPORT
 std::vector<ossia::net::node_base*>
 list_all_children(ossia::net::node_base* node);
+
+OSSIA_EXPORT
+std::vector<std::tuple<double, std::string, ossia::net::node_base*>>
```

Comment:
I'd be in favour of a small struct instead of a tuple so that we know without any doubt what everything is

Reply:
fixed in f1204341